examples/NewRelic.LogEnrichers.Log4Net.Examples/NewRelic.LogEnrichers.Log4Net.Examples.csproj

```diff
@@ -1,8 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
Not sure exactly what Snyk is or how you're using it... but it's odd to me that it would recommend a Core version that's out-of-support and not receiving security fixes: https://dotnet.microsoft.com/platform/support/policy/dotnet-core

Not a big deal, though.

Reply:
Yeah, we're still running into gotchas with Synk.  I'm going to create a separate issue to upgrade all of the example apps to 3.1.